tldw_Server_API/app/api/v1/endpoints/config_info.py

```
@@ -406,3 +415,333 @@ async def get_quickstart_redirect():
+        valid, error = await asyncio.wait_for(
+            _validate_provider_http(provider, api_key, info),
+            timeout=_VALIDATION_TIMEOUT_SECONDS,
+        )
+        return ProviderValidateResponse(provider=provider, valid=valid, error=error)
+    except asyncio.TimeoutError:
+        return ProviderValidateResponse(
+            provider=provider,
+            valid=False,
+            error=f"Validation timed out after {_VALIDATION_TIMEOUT_SECONDS}s",
+        )
+    except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)